packages/@guardian/source-react-components/src/button/Button.tsx

```diff
@@ -38,13 +39,17 @@ export const Button = ({
 			iconSide,
 			nudgeIcon,
 			cssOverrides,
+			isLoading,
 		})}
 		type={type}
+		aria-live="polite"
+		aria-label={isLoading ? 'Loading' : undefined}
```

Comment:
I wonder if we want this to be configurable by the consumer? For instance, a consumer might want it to say "Loading profile data" or something more specific. It could default to "Loading"

```ts
loadingLabel: string = "Loading"
```

Reply:
nice suggestion, I'll add this in